packages/utils/src/github/issuereporter.ts

```diff
@@ -11,6 +11,101 @@ type GithubData = {
     repo: string;
 }
 
+/*
+ * Octokit/types doesn't define a type for the items in a search and
+ * doesn't export the type IssuesUpdateEndpoint.
+ * These are a copy/paste of what they have in their code.
+ * Any suggestion to do this in a better way?
```

Comment:
Is there an issue opened reporting this? If so can you put the link in here?

Reply:
No, but I can open it.

Reply:
Now the types looks better hehehe. They say that in 2-3 months they will have names for the types.